content/applications/sales/crm/pipeline/merge_similar.rst

```diff
@@ -40,14 +41,15 @@ Merging similar leads and opportunities
    of which record was created first.
 
 After confirming that the leads/opportunities should be merged, return to the Kanban view using the
-breadcrumb link, or by clicking the :guilabel:`Similar Leads` smart button. Click the :guilabel:`≣
-(view list)` icon to change to list view.
+breadcrumb link, or by clicking the :guilabel:`Similar Leads` smart button. Click the
+:icon:`oi-view-list` :guilabel:`(list)` icon to change to list view.
 
 Check the box on the left of the page for the leads/opportunities to be merged. Then, click the
-:guilabel:`⚙️ Actions` icon at the top of the page, to reveal a drop-down menu. From that drop-down
-menu, select the :guilabel:`Merge` option to merge the selected opportunities or leads.
+:icon:`fa-cog` :guilabel:`(Actions)` icon at the top of the page, to reveal a drop-down menu. From
```

Comment:
In this instance, "Actions" text is visible in the button as well :)

![Screenshot 2024-06-28 at 12 27 45 PM](https://github.com/odoo/documentation/assets/36018073/0aa3da56-feb9-4a90-8a73-b042b25986d7)

```suggestion
:icon:`fa-cog` :guilabel:`Actions` icon at the top of the page, to reveal a drop-down menu. From
```